### PR TITLE
cloud: July 11 release notes

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

July 11 release notes

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 1]
```diff
@@ -8,6 +8,34 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+- Introduce TiDB Bot (beta), an OpenAI-powered chatbot that offers multi-language support, 24/7 real-time response, and integrated documentation access.
+
+    TiDB Bot provides you with the following benefits:
+
+    - Continuous support: Always available to assist and answer your questions for an enhanced support experience.
+    - Improved efficiency: Automated responses reduce latency, improving overall operations.
+    - Seamless documentation access: Direct access to TiDB Cloud documentation for easy information retrieval and quick issue resolution.
+
+    To use TiDB Bot, click **?** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com), and select **Ask TiDB Bot** to start a chat.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-release-notes.md:25]:
```suggestion
  To use TiDB Bot, click **?** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com), and select **Ask TiDB Bot** to start a chat.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 2]
```diff
@@ -8,6 +8,32 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+- [TiDB Serverless](/tidb-cloud/select-cluster-tier.md#tidb-serverless) now is Generally Available.
+
+- Introduce TiDB Bot (beta), an OpenAI-powered chatbot that offers multi-language support, 24/7 real-time response, and integrated documentation access.
+
+    TiDB Bot provides you with the following benefits:
+
+    - Continuous support: Always available to assist and answer your questions for an enhanced support experience.
+    - Improved efficiency: Automated responses reduce latency, improving overall operations.
+    - Seamless documentation access: Direct access to TiDB Cloud documentation for easy information retrieval and quick issue resolution.
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-release-notes.md:23]: [ref](https://developers.google.cn/style/colons#colons-within-sentences): In general, the first word in the text that follows a colon should be in lowercase.
```suggestion
    - Continuous support: always available to assist and answer your questions for an enhanced support experience.
    - Improved efficiency: automated responses reduce latency, improving overall operations.
    - Seamless documentation access: direct access to TiDB Cloud documentation for easy information retrieval and quick issue resolution.
```

**Review by ran-huang — APPROVED** [review]
rest LGTM

**Review by qiancai — APPROVED** [review]
/lgtm

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**